src/happy_vllm/logits_processors/response_pool.py

```diff
@@ -1,103 +0,0 @@
-#!/usr/bin/env python3
-# Copyright (C) <2018-2024>  <Agence Data Services, DSI France Travail>
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-
-import math
-import torch
-from typing import List
-
-from happy_vllm import utils
-
-
-class VLLMLogitsProcessorResponsePool:
```

Comment:
This class is still used in examples/function_tools/functional.py (import + one other instance)